python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -225,6 +235,11 @@ def to_batches(tensor):
                 "vf_explained_var": explained_variance(
                     tf.reshape(self.loss.vtrace_returns.vs, [-1]),
                     tf.reshape(to_batches(values)[:-1], [-1])),
+                "KLs": self.KLs,
```

Comment:
Let's drop self.KLs
```suggestion
```

Reply:
self.KLs was used to create the histograms on tensorboard. is that something we're still interested in?